src/settings_internal.js

```diff
@@ -75,6 +75,8 @@ var MAX_GLOBAL_ALIGN = -1;
 // List of functions implemented in compiled code; received from the backend.
 var IMPLEMENTED_FUNCTIONS = [];
 
+var WEAK_DECLARES = [];
```

Comment:
is this fastcomp-only, and so something we should remove with fastcomp? (if so a comment might be good?)